features/admin.applications.v1/components/edit-application.tsx

```diff
@@ -1080,160 +529,760 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
+     * @returns The rendered tab pane.
+     */
+    const MarkdownGuideTabPane = (guideContent: string): ReactElement => {
+        const firstProtocolName: string = mapProtocolTypeToName(application?.inboundProtocols?.[0]?.type);
+
+        return (
+            <ResourceTab.Pane controlledSegmentation>
+                <ApplicationMarkdownGuide
+                    application={ application }
+                    inboundProtocolConfigurations={ inboundProtocolConfig?.[firstProtocolName] }
+                    content={ guideContent }
+                    isLoading={ isLoading }
+                    protocolName={ firstProtocolName }
+                />
+            </ResourceTab.Pane>
+        );
+    };
+
+    /**
+     * Resolves the tab panes based on the application config.
+     *
+     * @returns Resolved tab panes.
+     */
+    const resolveTabPanes = (): ResourceTabPaneInterface[] => {
+        const panes: ResourceTabPaneInterface[] = [];
+        const extensionPanes: ResourceTabPaneInterface[] = [];
+
+        if (!tabPaneExtensions && applicationConfig.editApplication.extendTabs
+            && application?.templateId !== ApplicationManagementConstants.CUSTOM_APPLICATION_OIDC
+            && application?.templateId !== ApplicationManagementConstants.CUSTOM_APPLICATION_PASSIVE_STS
+            && application?.templateId !== ApplicationManagementConstants.CUSTOM_APPLICATION_SAML) {
+            return [];
+        }
+
+        if (tabPaneExtensions && tabPaneExtensions?.length > 0) {
+            extensionPanes.push(...cloneDeep(tabPaneExtensions));
+        }
+
+        if (featureConfig) {
+            if (isMyAccount) {
+                panes.push({
+                    componentId: "overview",
+                    menuItem: t("applications:myaccount.overview.tabName"),
+                    render: MyAccountOverviewTabPane
+                });
+            }
+            if (isFeatureEnabled(featureConfig?.applications,
+                ApplicationManagementConstants.FEATURE_DICTIONARY.get("APPLICATION_EDIT_GENERAL_SETTINGS"))
+                && (isSubOrganization() ?
+                    !brandingDisabledFeatures.includes(
+                        BrandingPreferencesConstants.APP_WISE_BRANDING_FEATURE_TAG): true)
+                && !isMyAccount
+            ) {
+                if (applicationConfig.editApplication.
+                    isTabEnabledForApp(
+                        inboundProtocolConfig?.oidc?.clientId,
+                        ApplicationTabTypes.GENERAL,
+                        tenantDomain
+                    )) {
+                    panes.push({
+                        componentId: "general",
+                        "data-tabid": ApplicationTabIDs.GENERAL,
+                        menuItem:
+                                 <Menu.Item data-tourid="general">
+                                     { t("applications:edit.sections.general.tabName") }
+                                 </Menu.Item>,
```

Comment:
```suggestion
                            <Menu.Item data-tourid="general">
                                { t("applications:edit.sections.general.tabName") }
                            </Menu.Item>,
```

Reply:
This is an existing code block. https://github.com/wso2/identity-apps/blob/05605b9367987c95ce06add63ce0e95494a90b2d/features/admin.applications.v1/components/edit-application.tsx#L269-L271